cargo/cargo_build_script_runner/lib.rs

```diff
@@ -103,12 +103,17 @@ impl BuildScriptOutput {
     /// Converts a [BufReader] into a vector of [BuildScriptOutput] enums.
     fn outputs_from_reader<T: Read>(mut reader: BufReader<T>) -> Vec<BuildScriptOutput> {
         let mut result = Vec::<BuildScriptOutput>::new();
-        let mut line = String::new();
-        while reader.read_line(&mut line).expect("Cannot read line") != 0 {
-            if let Some(bso) = BuildScriptOutput::new(&line) {
+        let mut buf = Vec::new();
+        while reader
+            .read_until(b'\n', &mut buf)
+            .expect("Cannot read line")
+            != 0
+        {
+            let line = &String::from_utf8_lossy(&buf);
```

Comment:
It looks like `cargo` actually completely skips any line which isn't `utf-8`: https://github.com/rust-lang/cargo/blob/0b84a35c2c7d70df4875a03eb19084b0e7a543ef/src/cargo/core/compiler/custom_build.rs#L566-L569

Can you do the same, rather than trying to parse the line?

Reply:
Done